paper/src/main/kotlin/org/empirewar/lemonadestand/paper/LemonadeStandPaper.kt

```diff
@@ -0,0 +1,148 @@
+package org.empirewar.lemonadestand.paper
+
+import org.bukkit.Bukkit
+import org.bukkit.OfflinePlayer
+import org.bukkit.event.EventHandler
+import org.bukkit.event.Listener
+import org.bukkit.plugin.java.JavaPlugin
+import org.empirewar.lemonadestand.LemonadeStand
+import org.empirewar.lemonadestand.WebServer
+import org.empirewar.lemonadestand.discord.WebhookSender
+import org.empirewar.lemonadestand.event.EventCaller
+import org.empirewar.lemonadestand.logging.PrettyFormatter
+import org.empirewar.lemonadestand.paper.event.KoFiTransactionEvent
+import org.empirewar.lemonadestand.paper.event.PaperEventCaller
+import org.empirewar.lemonadestand.paper.scheduler.PaperScheduler
+import org.empirewar.lemonadestand.scheduler.PlatformScheduler
+import org.spongepowered.configurate.CommentedConfigurationNode
+import org.spongepowered.configurate.ConfigurationNode
+import org.spongepowered.configurate.loader.ConfigurationLoader
+import org.spongepowered.configurate.yaml.YamlConfigurationLoader
+import java.io.File
+import java.io.IOException
+import java.net.MalformedURLException
+import java.util.logging.FileHandler
+import java.util.logging.Level
+import java.util.logging.Logger
+
+class LemonadeStandPaper: JavaPlugin(), LemonadeStand<OfflinePlayer>, Listener {
+
+    lateinit var transactionLogger: Logger
+        private set
```

Comment:
Why this weird formatting?

Reply:
I think you asked this before, iirc it's just weird kotlin, I think private set generates a setter, which we don't even need